xbmc/addons/kodi-dev-kit/include/kodi/versions.h

```diff
@@ -37,7 +37,7 @@
 // because cmake uses this area in this form to perform its addon dependency
 // check.
 // clang-format off
-#define ADDON_GLOBAL_VERSION_MAIN                     "1.3.0"
+#define ADDON_GLOBAL_VERSION_MAIN                     "1.3.1"
```

Comment:
Why is this bump needed?

Reply:
Increase is related to the API changes but as `ADDON_GLOBAL_VERSION_MAIN_MIN` not changed, does all addons still works with it.

Reply:
Semantically, you can bump the patch version when you add a period, but the min version is where you start worrying about compatibility